traffic_portal/app/src/common/api/DeliveryServiceUriSigningKeysService.js

```diff
@@ -17,17 +17,31 @@
  * under the License.
  */
 
-var DeliveryServiceUriSigningKeysService = function(Restangular) {
+var DeliveryServiceUriSigningKeysService = function($http, ENV) {
 
 	this.getKeys = function(id) {
-		return Restangular.one("deliveryservices", id).one("urisignkeys").get();
```

Comment:
TypeError: keys.plain is not a function
    at new DeliveryServiceUriSigningKeysController (DeliveryServiceUriSigningKeysController.js:23)